### ISSUE TITLE
AsyncIterator のエラーがでます、なんでなんでしょうか、、

### ISSUE THREAD

**Issue by yosuke1985** [Quramy/ionic-apollo-simple-app#1]
![screen shot 2017-08-16 at 17 53 45](https://user-images.githubusercontent.com/7687894/29355413-e9f3301c-82ab-11e7-9e45-8d76ffb88718.png)

こんなエラーがionic serveを打ち込むと出ます

**Comment by ntziolis** [comment]
Open tsconfig.json and replace:
```     
      "dom",
      "es2015"
```

Replace with
```     
      "dom",
      "es2015",
      "esnext.asynciterable"
```
I have provided a fixed this and provided a pull request here: https://github.com/Quramy/ionic-apollo-simple-app/pull/2

**Comment by Quramy** [comment]
closed via #2
